docs/sources/enterprise/access-control-refactor/provision-custom-roles/assigning-custom-roles.md

```diff
@@ -0,0 +1,165 @@
+
+| Attribute      | Description                                                                                                                                                                                                                                                  |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `name`         | Enter the name of the custom role.                                                                                                                                                                                                                           |
+| `version`      | Enter the custom role version number. Assignments are updated if the version of the role is greater than or equal to the version number stored internally. If you are updating a role assignment, you are not required to increment the role version number. |
+| `orgId`        | If you do not enter an `orgId`, it inherits the `orgId` from `role`. For global roles the default `orgId` is used. `orgId` in the `role` and in the assignment must be the same for non-global roles.                                                        |
+| `permissions`  | Enter the permissions `action` and `scope` values. For more information about permissions actions and scopes, refer to [LINK]                                                                                                                                |
+| `builtInRoles` | Enter the `name` of an organization role, for example `Viewer`, `Editor`, or `Admin`, or enter `Grafana Admin`.                                                                                                                                              |
+
+1. Reload the provisioning configuration file.
+
+   For more information about reloading the provisioning configuration at runtime, refer to [Reload provisioning configurations]({{< relref "../../http_api/admin/#reload-provisioning-configurations" >}}).
+
+The following example assigns the `users:editor` custom role to the built in editor and admin roles.
+
+```yaml
+# config file version
+apiVersion: 1
+
+# Roles to insert/update in the database
+roles:
+  - name: custom:users:editor
+    description: 'This role allows users to list/create/update other users in the organization'
+    version: 1
+    orgId: 1
+    permissions:
+      - action: 'users:read'
+        scope: 'users:*'
+      - action: 'users:write'
+        scope: 'users:*'
+      - action: 'users:create'
+        scope: 'users:*'
+    builtInRoles:
+      - name: 'Editor'
+      - name: 'Admin'
+```
+
+## Assign a custom role to a team
+
+Need information here on why a user would want to complete this task. What are the benefits?
```

Comment:
same as above.I'm guessing you put these questions in for the review? that's good to know in that case that we can work like that!